src/InstallerHosts/Elastic.InstallerHosts/Elasticsearch/Tasks/Install/SetupXPackPasswordsTask.cs

```diff
@@ -111,5 +119,63 @@ private void SetPassword(HttpClient client, string elasticUserPassword, string u
 			}
 			this.Session.SendProgress(200, $"Changed password for user '{user}'");
 		}
+
+		// Might be moved and refactored when is needed for other tasks
```

Comment:
```suggestion
		// TODO: Might be moved and refactored when is needed for other tasks
```

Use todo syntax so it shows up in Todo explorer.